registry_module_integration_test.go

```diff
@@ -144,6 +144,26 @@ func TestRegistryModulesCreate(t *testing.T) {
 
 			assertRegistryModuleAttributes(t, rm)
 		})
+
+		t.Run("with no-code attribute", func(t *testing.T) {
+			skipIfBeta(t)
+			options := RegistryModuleCreateOptions{
+				Name:         String("iam"),
+				Provider:     String("aws"),
+				NoCode:       true,
+				RegistryName: PrivateRegistry,
+			}
+			rm, err := client.RegistryModules.Create(ctx, orgTest.Name, options)
+			require.NoError(t, err)
+			assert.NotEmpty(t, rm.ID)
+			assert.Equal(t, *options.Name, rm.Name)
+			assert.Equal(t, *options.Provider, rm.Provider)
+			assert.Equal(t, options.RegistryName, rm.RegistryName)
+			assert.Equal(t, orgTest.Name, rm.Namespace)
+			assert.Equal(t, options.NoCode, rm.NoCode)
```

Comment:
what do you think about updating already existing `create RegistryModules` tests to assert that `rm.NoCode` is `false` which would be the default value. This would capture cases where NoCode is not set at creation.

Reply:
Hi @Uk1288 thanks for reviewing this PR! I went with your suggestion 👍🏼